x-pack/plugins/apm/public/components/shared/charts/timeseries_chart.tsx

```diff
@@ -59,6 +59,7 @@ interface Props {
   anomalyTimeseries?: ReturnType<
     typeof getLatencyChartSelector
   >['anomalyTimeseries'];
+  customTheme?: Record<string, unknown>;
```

Comment:
`customTheme` looks unused. I think it's better to remove it than adding it prematurely

Reply:
nvm - I see it's used now.